### PR TITLE
export import: document the import export tool

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

File: content/uc-doc/administration/import_export.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+If you already have a system with your users or even other resources configured you can use it as
+the base to fill up your import file.
+
+For example, if you have a CSV file with your users in it, you can modify the header of you CSV to
+match the available fields in the export file and use the CSV to fill you export file with the
+following command
+
+```sh
+cat user.csv | wazo-generate-dump import --users my-export.ods
```
Comment (fblackburn1) [inline content/uc-doc/administration/import_export.md:81]: Not 100% sure
```suggestion
cat user.csv | wazo-generate-dump add --users my-export.ods
```

**Review by fblackburn1 — COMMENTED** [review]
Should we disable DEBUG logger by default?
```
08:50 $ wazo-generate-dump list resources
2021-07-01 08:50:54,760 (DEBUG) (lml.plugin): declare 'pyexcel-io reader' plugin manager
2021-07-01 08:50:54,760 (DEBUG) (lml.plugin): declare 'pyexcel-io writer' plugin manager
2021-07-01 08:50:54,760 (DEBUG) (lml.plugin): declare 'pyexcel-io v2 writer' plugin manager
2021-07-01 08:50:54,760 (DEBUG) (lml.plugin): declare 'pyexcel-io v2 reader' plugin manager
2021-07-01 08:50:54,762 (DEBUG) (lml.loader): scanning for plugins...
2021-07-01 08:50:54,768 (DEBUG) (lml.loader): ignored pyexcel_io
2021-07-01 08:50:54,768 (DEBUG) (lml.utils): found pyexcel_ods
2021-07-01 08:50:54,770 (DEBUG) (lml.utils): found pyexcel_io.readers
2021-07-01 08:50:54,770 (DEBUG) (lml.utils): found pyexcel_io.writers
2021-07-01 08:50:54,770 (DEBUG) (lml.utils): found pyexcel_io.database
2021-07-01 08:50:54,770 (DEBUG) (lml.loader): scanning done
contexts users lines ring_groups extensions group_members voicemails incalls schedules schedule_times function_keys
```

**Review by mwolff44 — CHANGES_REQUESTED** [review]

File: content/uc-doc/administration/import_export.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+Installing the import tool is required on the exporting and importing system.
+
+To install it on a debian based system uses the following commands on the command line.
+
+```sh
+apt update && apt install git python3-pip
+git clone https://github.com/wazo-platform/wazo-export-import.git
+cd wazo-export-import
+pip3 install -rrequirements.txt
```
[the remark this answers is not in the document]
Comment (mwolff44) [reply content/uc-doc/administration/import_export.md:26]: also change -rrequirements to -r requirements

[the remark this answers is not in the document]
Comment (pc-m) [reply content/uc-doc/administration/import_export.md:26]: Do you still have the error you got on python 3.5? I would rather fix and not mess have to intall python 3.7

**Comment by pc-m** [comment]
> Should we disable DEBUG logger by default?
> 
> ```
> 08:50 $ wazo-generate-dump list resources
> 2021-07-01 08:50:54,760 (DEBUG) (lml.plugin): declare 'pyexcel-io reader' plugin manager
> 2021-07-01 08:50:54,760 (DEBUG) (lml.plugin): declare 'pyexcel-io writer' plugin manager
> 2021-07-01 08:50:54,760 (DEBUG) (lml.plugin): declare 'pyexcel-io v2 writer' plugin manager
> 2021-07-01 08:50:54,760 (DEBUG) (lml.plugin): declare 'pyexcel-io v2 reader' plugin manager
> 2021-07-01 08:50:54,762 (DEBUG) (lml.loader): scanning for plugins...
> 2021-07-01 08:50:54,768 (DEBUG) (lml.loader): ignored pyexcel_io
> 2021-07-01 08:50:54,768 (DEBUG) (lml.utils): found pyexcel_ods
> 2021-07-01 08:50:54,770 (DEBUG) (lml.utils): found pyexcel_io.readers
> 2021-07-01 08:50:54,770 (DEBUG) (lml.utils): found pyexcel_io.writers
> 2021-07-01 08:50:54,770 (DEBUG) (lml.utils): found pyexcel_io.database
> 2021-07-01 08:50:54,770 (DEBUG) (lml.loader): scanning done
> contexts users lines ring_groups extensions group_members voicemails incalls schedules schedule_times function_keys
> ```

That's weird, I do not have these debugs.

I want to be able to see debugs when using `--debug` but I do not want to see debugs from `lml.*` which I don't at the moment

File: content/uc-doc/administration/import_export.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (fblackburn1) [reply content/uc-doc/administration/import_export.md]: Supporting more debian version can be added in another PR. I added a comment to explicitly show that installation is only tested on debian buster

**Comment by fblackburn1** [comment]
> That's weird, I do not have these debugs.
> 
> I want to be able to see debugs when using `--debug` but I do not want to see debugs from `lml.*` which I don't at the moment

I tested on ubuntu with virtualenv only (not on debian vanilla). Anyways is not related to documentation

**Review by fblackburn1 — APPROVED** [review]

File: content/uc-doc/administration/import_export.md [hunk 3]
```diff
@@ -20,6 +20,7 @@ Installing the import tool is required on the exporting and importing system.
 To install it on a debian based system uses the following commands on the command line.
 
 ```sh
+# Debian Buster (10)
```
Comment (mwolff44) [inline content/uc-doc/administration/import_export.md:23]: and also python version, as you can have debian 10 with python3.5 (migration from debian 9)

**Review by mwolff44 — CHANGES_REQUESTED** [review]

File: content/uc-doc/administration/import_export.md (hunk 3 above)
Comment (fblackburn1) [reply content/uc-doc/administration/import_export.md:23]: I never saw a debian Buster without python3.7, even with migration from Stretch
I am not sure it possible to have Buster without python3.7 ....

**Comment by fblackburn1** [comment]
regate

**Pull request merged**